BandNameGenerator/Genres/Genre.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BandNameGenerator.Genres
+{
+    public abstract class Genre
+    {
+        private readonly List<Formula> formulas;
+
+        public Genre()
+        {
+            formulas = new List<Formula>
+            {
+                SharedFormulas.ThePluralNoun,
+                SharedFormulas.TheAdjectiveSingularNoun,
+                SharedFormulas.TheAdjectivePluralNoun,
+                SharedFormulas.AdjectiveSingularNoun,
+                SharedFormulas.AdjectivePluralNoun,
+                SharedFormulas.SingularNounPossessiveSingularNoun
+            };
+
+            formulas.AddRange(GenreSpecificFormulas);
+        }
+
+        public string[] SingularNouns { get; protected set; } = new string[] { };
+        public string[] PluralNouns { get; protected set; } = new string[] { };
+        public string[] Adjectives { get; protected set; } = new string[] { };
+        public string[] ThirdPersonSingularVerbs { get; protected set; } = new string[] { };
+        public string[] ThirdPersonPluralVerbs { get; protected set; } = new string[] { };
+        public string[] TransitiveVerbs { get; protected set; } = new string[] { };
+
+        protected List<Formula> GenreSpecificFormulas { get; set; } = new List<Formula>();
```

Comment:
Nothing wrong here. But an idea I'm kicking around is, you could make `formulas` into a property, and child classes could do `Formulas.AddRange(...)` to add their genre-specific items. This would also allow child classes to remove standard formulas, which may be good or bad depending on how you intend it to be used.

Reply:
Per discussion, decided to convert to a method so child classes can't remove formulas.